packages/terra-form-select/src/SingleSelectField.jsx

```diff
@@ -61,6 +71,7 @@ const propTypes = {
    */
   maxWidth: PropTypes.string,
   /**
+   * @private
```

Comment:
why `onChange()` has been marked as private ..??

Reply:
Suggestion under form-select [epic](https://jira2.cerner.com/browse/UXPLATFORM-6648) states "**nothing unexpected should ever happen without explicitly warning the user**" . Moved it to private to avoid using handlers upon selection